README.md

```diff
@@ -161,80 +161,132 @@ These instructions assume the release has been deployed to bosh-lite
 	
 ### Using Diego API
 
-#### DesiredLRPCreateRequest
-In order to receive TCP traffic on a given application port, the [DesiredLRPCreateRequest] (https://github.com/cloudfoundry-incubator/receptor/blob/master/doc/lrps.md#describing-desiredlrps) should be created as follows:
+The following example starts [the Redis Docker image](https://registry.hub.docker.com/_/redis/) as an LRP, and automatically configures the TCP Router to route traffic for a requested external port on the router to the Redis process. We will be using the Diego Receptor API; for more information see [Diego API Docs](https://github.com/cloudfoundry-incubator/receptor/tree/master/doc). 
 
-```
-{
-    "process_guid": "some-guid",
-    "domain": "some-domain",
-
-    "instances": 17,
-
-    "rootfs": "VALID-ROOTFS",
```

Comment:
@shalako Would it make sense to provide the currently supported rootFs's?

Notes: 
For Bosh-Lite: "preloaded:cflinuxfs2:"
For Lattice: "docker:///cloudfoundry/trusty64"